.prettierrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require('eslint-config-brainly/prettier.config');
```

Comment:
Please ignore that, it satisfies vscode when prettier formatting is disabled for projects not having config file.